fs/fs.go

```diff
@@ -78,9 +78,10 @@ func NewFsInfo(context Context) (FsInfo, error) {
 	}
 	supportedFsType := map[string]bool{
 		// all ext systems are checked through prefix.
-		"btrfs": true,
-		"xfs":   true,
-		"zfs":   true,
+		"btrfs":   true,
+		"overlay": true,
```

Comment:
Ideally docker root fs should be detected using docker's APIs (info) and a statfs on that should be good enough. 
This code just needs some love.
This logic here was originally meant for discovering filesystems of interest at a global level. 
Layered filesystems that are ephemeral are not of interest to most users in the global context.